srcpkgs/gnome-tweaks/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'gnome-tweaks'
 pkgname=gnome-tweaks
-version=45.2
+version=46.1
 revision=1
 build_style=meson
-hostmakedepends="pkg-config gettext"
-makedepends="libhandy1-devel"
-depends="gtk+3 dconf gnome-settings-daemon mutter libnotify python3-gobject libhandy1"
+build_helper=gir
```

Comment:
Not sure this is necessary. (I don't see any GIR/Typelib generation)
Let me know if it is.